### PR TITLE
Update restplus

### PR DESCRIPTION
**Pull request by noirbizarre**
This PR upgrades to latest Flask-RESTPlus (0.9.2) and apply some fixes

### CODE REVIEW

File: udata/api/commands.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 from udata.api import api
 from udata.commands import submanager
 
-from flask.ext.restplus.swagger import Swagger
+# from flask.ext.restplus.swagger import Swagger
```
Comment (davidbgk) [inline udata/api/commands.py]: Remove

File: udata/api/commands.py [hunk 2]
```diff
@@ -21,9 +21,22 @@
 
 
 @m.option('-p', '--pretty', action='store_true', default=False,
-          help='Pretty print (sort')
+          help='Pretty print')
 def swagger(pretty):
     '''Dump the swagger specifications'''
-    data = Swagger(api).as_dict()
+    # data = Swagger(api).as_dict()
```
Comment (davidbgk) [inline udata/api/commands.py]: Remove

File: udata/api/commands.py [hunk 3]
```diff
@@ -21,9 +21,22 @@
...
     kwargs = dict(indent=4) if pretty else {}
+    print(json.dumps(api.__schema__, **kwargs))
```
Comment (davidbgk) [inline udata/api/commands.py]: `print`?

File: udata/tests/api/test_auth_api.py [hunk 4]
```diff
@@ -109,8 +108,7 @@ def test_invalid_apikey(self):
 
         self.assert401(response)
         self.assertEqual(response.content_type, 'application/json')
-        self.assertEqual(response.json,
-                         {'status': 401, 'message': 'Invalid API Key'})
+        self.assertIn('message', response.json)
```
Comment (davidbgk) [inline udata/tests/api/test_auth_api.py]: Doesn't test anymore the content of the error, is that intentional?

File: udata/tests/api/test_auth_api.py [hunk 5]
```diff
@@ -120,8 +118,7 @@ def test_inactive_user(self):
 
         self.assert401(response)
         self.assertEqual(response.content_type, 'application/json')
-        self.assertEqual(response.json,
-                         {'status': 401, 'message': 'Inactive user'})
+        self.assertIn('message', response.json)
```
Comment (davidbgk) [inline udata/tests/api/test_auth_api.py]: Same here.

File: udata/tests/api/test_auth_api.py [hunk 6]
```diff
@@ -223,7 +216,4 @@ def get(self):
         response = self.get(url_for('api.exception'))
 
         self.assert403(response)
-        self.assertEqual(response.json, {
-            'message': 'Permission denied',
-            'status': 403
-        })
+        self.assertIn('message', response.json)
```
Comment (davidbgk) [inline udata/tests/api/test_auth_api.py]: Same here.

File: udata/tests/api/test_oembed_api.py [hunk 7]
```diff
@@ -57,10 +57,7 @@ def test_oembeds_dataset_api_get_without_references(self):
         response = self.get(url_for('api.oembeds'))
         self.assert400(response)
         data = json.loads(response.data)
-        self.assertEqual(
-            data['message']['references'],
-            ("(References of the resources to embed.)  "
-             "Missing required parameter in the query string"))
+        self.assertIn('references', data['errors'])
```
Comment (davidbgk) [inline udata/tests/api/test_oembed_api.py]: Same here.

File: udata/api/__init__.py [hunk 8]
```diff
@@ -45,8 +45,8 @@ def __init__(self, app=None, **kwargs):
         self.authorizations = {
             'apikey': {
                 'type': 'apiKey',
-                'passAs': 'header',
-                'keyname': HEADER_API_KEY
+                'in': 'header',
+                'name': HEADER_API_KEY
```
Comment (davidbgk) [inline udata/api/__init__.py]: Is that change and the one just below backward-compatible for existing API calls?

File: udata/api/__init__.py [hunk 9]
```diff
@@ -68,15 +68,15 @@ def wrapper(func):
 
     def _apply_secure(self, func, permission=None):
         '''Enforce authentication on a given method/verb'''
-        self._handle_api_doc(func, {'authorizations': 'apikey'})
+        self._handle_api_doc(func, {'security': 'apikey'})
```
Comment (davidbgk) [inline udata/api/__init__.py]: I mean that one ;-)

File: udata/api/commands.py (hunk 3 above)
Comment (noirbizarre) [inline udata/api/commands.py]: yes, it's for output on stdout. If you have a better alternative, I'll take it.

File: udata/tests/api/test_auth_api.py (hunk 4 above)
Comment (noirbizarre) [inline udata/tests/api/test_auth_api.py]: Yes, standards error messages are handled upstream by Flask-RESTPlus, it's not relevant to test them here and they are subject to change on each upgrade

File: udata/api/__init__.py (hunk 8 above)
Comment (noirbizarre) [inline udata/api/__init__.py]: It's only used in Swagger specifications. It wasn't working before (it was expressed for Swagger 1.x). Now it is (used in Swagger API doc only)

Comment (davidbgk) [inline udata/api/__init__.py]: OK :)

File: requirements/install.pip [hunk 10]
```diff
@@ -40,6 +40,3 @@ wtforms-json==0.2.10
 wtforms==2.1
 xmltodict
 geojson>=1.3.1
-
-# Pin this until flask-restplus works with 0.3.4
-Flask-RESTful==0.3.3
```
Comment (davidbgk) [inline requirements/install.pip]: Oh I didn't noticed that, each removed dependency is :champagne: :tada:

File: udata/api/commands.py (hunk 3 above)
Comment (davidbgk) [inline udata/api/commands.py]: How do you consume it?

File: requirements/install.pip (hunk 10 above)
Comment (noirbizarre) [inline requirements/install.pip]: :+1:

**Pull request merged**